scripts/ci-build.sh

```diff
@@ -296,7 +296,10 @@ stage_artifact mv gdp-src-build/tmp/deploy/licenses/genivi-dev-platform*/license
 stage_artifact mv gdp-src-build/tmp/deploy/sdk*
 stage_artifact cp gdp-src-build/tmp/deploy/images/*
 stage_artifact cp gdp-src-build/conf/*.conf
-stage_artifact cp logs.tar.gz
+stage_artifact mv logs.tar.gz
+stage_artifact cp gdp-src-build/tmp/buildhistory/images/*/glibc/genivi-dev-platform/files-in-image.txt
+stage_artifact mv gdp-src-build/tmp/buildhistory
```

Comment:
Agree that many might not find the files buried in buildhistory. Especially as you have to go through several folder levels to find any files. Just a small suggestion if you return to this in the future - realising you probably have more important things to do than forever finesse this PR. Maybe consider having a symlink to the files file in buildhistory rather than a copy.